app/models/user.rb

```diff
@@ -1,8 +1,11 @@
 class User < ApplicationRecord
   has_secure_password
   has_many :orders
+
+  validates :name, presence: true
   validates :username, presence: true,
                        uniqueness: true
+  validates :password, presence: true
```

Comment:
@NateAnderson1780 You can just make a new commit to your branch and push it up; it will just update this pull request.